internal/xds/bootstrap/bootstrap_test.go

```diff
@@ -432,6 +579,29 @@ func (s) TestGetConfiguration_Success(t *testing.T) {
 		{"goodBootstrap", configWithGoogleDefaultCredsAndV3},
 		{"multipleXDSServers", configWithMultipleServers},
 		{"serverSupportsIgnoreResourceDeletion", configWithGoogleDefaultCredsAndIgnoreResourceDeletion},
+		{"istioStyleInsecureWithoutCallCreds", configWithIstioStyleNoCallCreds},
```

Comment:
If this case isn't covered already, can you please add a test case to verify that the call creds are not read when the environment variable is unset?

Reply:
Good shout; I've added one at the bottom. The call creds are still going to get parsed in the current implementation (as we're just doing `json.Unmarshal(data, &server)` ) , but they won't be used as DialOptions